content/en/docs/languages/java/automatic/configuration.md

```diff
@@ -30,6 +30,23 @@ specific portions of the SDK & agent:
 - [Span limits](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#span-limits)
 - [Using SPI to further configure the SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#customizing-the-opentelemetry-sdk)
 
+### Enable Resource Providers that are disabled by default
```

Comment:
Do "Resource Providers" need to be capitalized here?

Reply:
the link above has "OpenTelemetry Resource and service name" - so it's consistent

Reply:
then probably the link above should be updated to resource? ;-)

Reply:
I hope I got the one you mean :smile:

Reply:
We can follow up with a capitalization pass if we like later. I suspect there's several things that would be changed across many docs.